Launch.cmd

```diff
@@ -4,4 +4,4 @@ set VisualBasicDesignTimeTargetsPath=%VisualStudioXamlRulesDir%Microsoft.VisualB
 set FSharpDesignTimeTargetsPath=%VisualStudioXamlRulesDir%Microsoft.FSharp.DesignTime.targets
 set CSharpDesignTimeTargetsPath=%VisualStudioXamlRulesDir%Microsoft.CSharp.DesignTime.targets
 
-devenv /rootsuffix ProjectSystem
\ No newline at end of file
+devenv /rootsuffix Exp
```

Comment:
Note, this changed here to have this passed in and respect ROOTSUFFIX: https://github.com/dotnet/project-system/pull/3600.